monoid-extras.cabal

```diff
@@ -45,3 +45,12 @@ library
                      FlexibleInstances,
                      MultiParamTypeClasses,
                      TypeOperators
+
+benchmark semi-direct-product
+  default-language:  Haskell2010
+  hs-source-dirs: benchmarks
+  main-is: SemiDirectProduct.hs
+  type: exitcode-stdio-1.0
+  build-depends: base          >= 4.3 &&  < 4.10
+               , criterion
+               , monoid-extras == 0.4.0.2
```

Comment:
Is there any reason the version number for `monoid-extras` needs to be so specific here?